unity/PingPongGameExample/PingPongGame/Assets/Scripts/NetTest.cs

```diff
@@ -59,6 +100,33 @@ public double ConnectAndDisconnect(string host, int port=3000)
     return ts.TotalMilliseconds;
   }
 
+  // Create a client and connect/disconnect on a partcular site.
+  public double ConnectAndDisconnect(Site site)
+  {
+    stopWatch.Reset();
+
+    if (httpClient == null)
+    {
+      // TODO: GetConnection to connect from a particular network interface endpoint
+      httpClient = new HttpClient();
+      httpClient.Timeout = new TimeSpan(0, 0, 5); // seconds
+    }
+    // The nature of this app specific GET API call is to expect some kind of
+    // stateless empty body return also 200 OK.
```

Comment:
It seems this would be affected by CPU on the server. The server has to serve the request, and produce and send the 200 response. The way the SDK Demo App works when ICMP ping isn't supported is to time how long it takes to open a TCP socket. This is may 10-15% slower than an actual ping. It would be interesting to compare timings for  all of these: 1) ICMP ping, 2) Open socket time, 3) Full HTTP request with 200 response.

Reply:
Yes. The understanding of ICMP ping has issues on some cloud providers (blocked). Of course, high performance ping isn't comparable the actual server API handler up the stack either (python, C, go, etc.) either. We could add ping as an option, though it could be merely measuring the load balancer QOS rules (which would be separately interesting too).

Reply:
Now added Ping option. Makes sense for PingPong.